storage/innobase/row/row0merge.cc

```diff
@@ -4462,6 +4689,10 @@ row_merge_build_indexes(
 	/* Now we have files containing index entries ready for
 	sorting and inserting. */
 
+	DBUG_EXECUTE_IF(
+		"ib_merge_wait_after_read",
```

Comment:
Please don't :) Luckily, this does not seem to have any associated testcase (something not imported from MariaDB?). Can it be removed, or if a testcase import is missed, replaced with something else?

Reply:
Test cases were cleaned up and use existing upstream DBUG_SYNC points